monitoring/src/contracts.ts

```diff
@@ -0,0 +1,42 @@
+import { context, Environment } from "./context"
+import type { Bridge, TBTCVault } from "@keep-network/tbtc-v2.ts/dist/src/chain"
+import { EthereumBridge, EthereumTBTCVault} from "@keep-network/tbtc-v2.ts"
+import { providers } from "ethers"
+
+const resolve = () => {
+  let packageName: string
+
+  switch (context.environment) {
+    case Environment.Mainnet: {
+      packageName = "@keep-network/tbtc-v2-mainnet"
+      break
+    }
+    case Environment.Testnet: {
+      packageName = "@keep-network/tbtc-v2-testnet"
+      break
```

Comment:
I am confused about this part. Are we referencing the smart contracts package? We are not publishing them under those names.

Reply:
Look at the `package.json`, those are aliases over specific package versions.